<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="diva" ignored="1" name="Default test(failed)" tests="2" failures="2" timestamp="24 Nov 2021 18:49:44 GMT" time="0.063" errors="0">
  <testcase name="tc3" time="0.015" classname="org.testing.Rerunfail">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testing.Rerunfail.tc3(Rerunfail.java:24)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- tc3 -->
  <testcase name="tc1" time="0.0" classname="org.testing.Rerunfail">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testing.Rerunfail.tc1(Rerunfail.java:10)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- tc1 -->
  <testcase name="tc2" time="0.0" classname="org.testing.Rerunfail">
    <ignored/>
  </testcase> <!-- tc2 -->
</testsuite> <!-- Default test(failed) -->
